src/utils/prepareTodos.tsx

```diff
@@ -0,0 +1,27 @@
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+export function prepareTodos(todos: Todo[], querry = '', filter = Filter.All): Todo[] {
+  let copy = [...todos];
```

Comment:
If you can write constant instead let use it prefer.
let worsens the readability and perception of the code during scaling